src/clj/fluree/db/flake.cljc

```diff
@@ -17,18 +17,18 @@
 
 (declare equiv-flake assoc-flake get-flake-val nth-flake)
 
-(def ^:const min-s iri/min-sid)
-(def ^:const max-s iri/max-sid)
-(def ^:const min-p iri/min-sid)
-(def ^:const max-p iri/max-sid)
-(def ^:const min-dt iri/min-sid)
-(def ^:const max-dt iri/max-sid)
-(def ^:const min-t 0)
-(def ^:const max-t util/max-long)
-(def ^:const min-op false)
-(def ^:const max-op true)
-(def ^:const min-meta util/min-integer)
-(def ^:const max-meta util/max-integer)
+(def min-s iri/min-sid)
+(def max-s iri/max-sid)
+(def min-p iri/min-sid)
+(def max-p iri/max-sid)
+(def min-dt iri/min-sid)
+(def max-dt iri/max-sid)
+(def min-t 0)
+(def max-t util/max-long)
+(def min-op false)
+(def max-op true)
+(def min-meta util/min-integer)
+(def max-meta util/max-integer)
```

Comment:
Do we want to remove all of them, or just the ones that interact with the `SID` class?

Reply:
good point. all add the non-sid ones back